app/src/androidTest/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/splash/SplashScreenTest.kt

```diff
@@ -1,32 +1,104 @@
 package com.kks.nimblesurveyjetpackcompose.ui.presentation.splash
 
-import androidx.compose.ui.test.assertIsDisplayed
-import androidx.compose.ui.test.junit4.createAndroidComposeRule
-import androidx.compose.ui.test.junit4.createComposeRule
-import androidx.compose.ui.test.onNodeWithContentDescription
+import androidx.compose.ui.test.*
 import androidx.test.ext.junit.runners.AndroidJUnit4
 import com.kks.nimblesurveyjetpackcompose.R
 import com.kks.nimblesurveyjetpackcompose.base.BaseAndroidComposeTest
-import com.kks.nimblesurveyjetpackcompose.ui.presentation.main.MainActivity
+import com.kks.nimblesurveyjetpackcompose.di.RepositoryModule
+import com.kks.nimblesurveyjetpackcompose.model.ResourceState
+import com.kks.nimblesurveyjetpackcompose.model.response.LoginResponse
+import com.kks.nimblesurveyjetpackcompose.repo.login.LoginRepo
 import com.kks.nimblesurveyjetpackcompose.ui.theme.NimbleSurveyJetpackComposeTheme
+import com.kks.nimblesurveyjetpackcompose.viewmodel.splash.SPLASH_TIME
+import com.kks.nimblesurveyjetpackcompose.viewmodel.splash.SplashViewModel
 import com.ramcosta.composedestinations.navigation.EmptyDestinationsNavigator
+import dagger.hilt.android.testing.BindValue
+import dagger.hilt.android.testing.HiltAndroidRule
+import dagger.hilt.android.testing.HiltAndroidTest
+import dagger.hilt.android.testing.UninstallModules
+import io.mockk.coEvery
+import io.mockk.mockk
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flowOf
+import org.junit.Before
 import org.junit.Rule
 import org.junit.Test
 import org.junit.runner.RunWith
 
-@RunWith(AndroidJUnit4::class)
+const val EMAIL = "example@gmail.com"
+const val VALID_PASSWORD = "valid"
+const val INVALID_PASSWORD = "invalid"
```

Comment:
Some unused `const`s here 👀 
```suggestion
const val VALID_PASSWORD = "valid"
```